Source/AST/Styling/Text Views/DownDebugTextView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DownDebugTextView.swift
+//  Down
+//
+//  Created by John Nguyen on 06.08.19.
+//  Copyright © 2019 Glazed Donut, LLC. All rights reserved.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+#elseif canImport(AppKit)
+
+import AppKit
+
+#endif
+
+/// A text view capable of parsing and rendering markdown via the AST, as well as line fragments.
+///
+/// See `DownDebugLayoutManager`.
+public class DownDebugTextView: DownTextView {
```

Comment:
We likely need some conditionals here and throughout the PR for `watchOS` since I don't think this will compile there. You can run the `watchOS` target from the dropdown in Xcode to see the failures

e.g.

```swift
#if os(watchOS)
    // Sorry, not available for watchOS
#else

...

#endif
```

Reply:
Good point, will do this.

Reply:
Done.